src/Azure.Functions.Cli/Common/DurableManager.cs

```diff
@@ -0,0 +1,180 @@
+﻿using System;
+using System.Linq;
+using System.Threading.Tasks;
+using Azure.Functions.Cli.Interfaces;
+using Colors.Net;
+using DurableTask.AzureStorage;
+using DurableTask.Core;
+using DurableTask.Core.History;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+using static Colors.Net.StringStaticMethods;
+
+namespace Azure.Functions.Cli.Common
+{
+    internal class DurableManager : IDurableManager
+    {
+        private readonly AzureStorageOrchestrationService _orchestrationService;
+
+        private readonly TaskHubClient _client;
+
+        private const string TaskHubName = "DurableFunctionsHub";
```

Comment:
Many customers will need to use a non-hardcoded task hub name. By default we should discover this from **host.json**.